From 4aaa48eb80c6802aec6d03e5695d2a0ff20e0fc2 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jpetridis@gnome.org>
Date: Thu, 24 Oct 2019 22:58:36 +0200
Subject: [PATCH] Drop the PangoFont find_shaper virtual method

This API has been removed from Pango 1.44.6, because it was completely
unused by anything.

However, PyGTK tries to bind everything, even unused API.

Removing this from PyGTK means we can build it against the latest Pango
again.

https://gitlab.gnome.org/GNOME/pango/issues/417
---
 pango.defs | 9 ---------
 1 file changed, 9 deletions(-)

diff --git pango.defs pango.defs
index 6935c964..da968f58 100644
--- a/pango.defs
+++ b/pango.defs
@@ -1391,15 +1391,6 @@
   )
 )
 
-(define-virtual find_shaper
-  (of-object "PangoFont")
-  (return-type "PangoEngineShape*")
-  (parameters
-    '("PangoLanguage*" "lang")
-    '("guint32" "ch")
-  )
-)
-
 (define-virtual get_glyph_extents
   (of-object "PangoFont")
   (return-type "none")
-- 
2.24.2

